test/data/stages/zipl/diff.json

```diff
@@ -1,12 +1,5 @@
 {
-  "added_files": [],
+  "added_files": [ "/etc/zipl.conf" ],
```

Comment:
How did this ever work?

Reply:
Good question! I am wondering the same.

Tests checkpoint the last stage of `a.json` and then run `b.json`. Effectively, this means that all generated files stay the same (rpm database, certificate caches, etc.) when running `b.json`, because only the last stage is run. 

However, `zipl` and `kernel-command-line` both had an empty stage in `a.json` (`"options": {}`), which broke that logic. Now both `a.json` and `b.json` were completely re-run and I saw large diffs from those generated files. Removing the empty stage made it work, but necessitated changing the diff in the above manner. I'm not sure who introduced these tests (@gicmo and @ondrejbudai?), but we probably want to find out if I'm not breaking anything before merging this.

Reply:
Very simple: The pipeline in `a.json` is changed now. Before, `a.json` had a zipl stage with no options and `b.json` then set the timeout. Now `a.json` does not have a zipl stage. I had the previous behaviour because you check two conditions, now we only check one.

Reply:
I missed the proper context, but the new commit messages cleared it up. Thanks. It is unfortunate and maybe the tests should be changed to make all the implicit assumptions more explicit. Or we ignore the files that are known to change. Either way, this is fine for now.